GameData/RealismOverhaul/RO_SuggestedMods/NearFutureTechnologies/NearFuturePropulsion/RO_NearFuture_Propulsion_Ion.cfg

```diff
@@ -0,0 +1,135 @@
+@PART[ionArgon-0625]:FOR[RealismOverhaul]
+{
+	//HERMeS/AEPS
+	%RSSROConfig = True
+	%engineType = HERMES
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = fxTransform
+		gimbalRange = 5
+		gimbalResponseSpeed = 2
+		useGimbalResponseSpeed = true
+	}
+}
+
+@PART[ionArgon-0625-2]:FOR[RealismOverhaul]
+{
+	//Gyro-Two Lensed Hall Effect Thruster
+	%RSSROConfig = True
+	%engineType = P5
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+
++PART[ionArgon-0625-2]:FOR[RealismOverhaul]
+{
+	// NASA-457M Hall Effect Thruster
+	%RSSROConfig = True
+	@name = RO-457M
+	%engineType = 457M
+
+	%rescaleFactor = 0.7312
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+
++PART[ionArgon-0625-2]:FOR[RealismOverhaul]
+{
+	// SPT-100 Hall Effect Thruster
+	// the 100 appears to stand for 100 mm
+	%RSSROConfig = True
+	@name = RO-SPT100
+	%engineType = SPT100
+
+	%rescaleFactor = 0.2
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+
++PART[ionArgon-0625]:FOR[RealismOverhaul]
+{
+	// PPS-1350 Hall Effect Thruster
+	%RSSROConfig = True
+	@name = RO-PPS1350
+	%engineType = PPS1350
+
+	%rescaleFactor = 0.5
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+
+@PART[ionArgon-0625-3]:FOR[RealismOverhaul]
+{
+	// NASA X3 Hall Effect Thruster
+	%RSSROConfig = True
+	%engineType = X3
+
+	%rescaleFactor = 1.28
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+
+@PART[ionXenon-0625]:FOR[RealismOverhaul]
+{
+	// NEXT Ion Thruster
+	%RSSROConfig = True
+	%engineType = NEXT
+
+	%rescaleFactor = 0.8
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = fxTransform
+		gimbalRange = 7
+ 		gimbalResponseSpeed = 2
+ 		useGimbalResponseSpeed = true
+	}
+}
++PART[ionXenon-0625]:FOR[RealismOverhaul]
+{
+	// ID-500 Gridded Ion Thruster
+	// Assuming 500 stands for 500 mm
+	%RSSROConfig = True
+	@name = RO-ID500
+	%engineType = ID500
+
+	%rescaleFactor = 1.14
+
+	!MODULE[TweakScale] {}
+	!EFFECTS[engage] {} // remove ignition sound effects
+}
+@PART[ionXenon-0625-2]:FOR[RealismOverhaul]
+{
+	// HiPEP Ion Thruster
+
+	%RSSROConfig = Test
```

Comment:
What is this RSSROConfig value?

Reply:
That’s what they were set to before. It marks the part as RO compatible, but adds a [TEST] tag to their names. This is a holdover from some ancient version of RO, and the new speculative tagging system should replace it.